README.md

```diff
@@ -52,19 +48,27 @@ to gain access to the latest features which have not been thoroughly tested yet.
     ```
 5. All done, you can open `http://localhost:3000` on your browser to view jetlog
 
+### Configuration
+
+| Name        | Default | Function            |
+|-------------|---------|---------------------|
+| PGID        | 1000    | Group ID for JetLog |
```

Comment:
> Group ID for Jetlog

I would like to keep the "Jetlog" spelling for consistency, since that's what I started out with. It would also be nice to have the default values enclosed in \`\`'s, so that they appear as code. The same goes for the lines below